dask/array/core.py

```diff
@@ -2273,6 +2273,10 @@ def concatenate(seq, axis=0, allow_unknown_chunksizes=False):
     stack
     """
     n = len(seq)
+
+    if n == 1:
+        return seq[0]
+
```

Comment:
Could see arguments either way. Put it after for now, but happy to move it up if that makes more sense.